tests/scripts/collect-logs.sh

```diff
@@ -20,6 +21,7 @@ NAMESPACES=("$CLUSTER_NAMESPACE")
 if [[ "$OPERATOR_NAMESPACE" != "$CLUSTER_NAMESPACE" ]]; then
   NAMESPACES+=("$OPERATOR_NAMESPACE")
 fi
+NAMESPACES+=("$KUBE_SYSTEM_NAMESPACE")
```

Comment:
Let's add the namespace to the list on line 20

Reply:
done